FABulous/fabric_generator/fabric_gen.py

```diff
@@ -978,24 +973,24 @@ def generateTile(self, tile: Tile) -> None:
         self.writer.addNewLine()
         self.writer.addLogicStart()
 
-        if tile.globalConfigBits > 0:
-            self.writer.addAssignScalar("FrameData_O_i", "FrameData_i")
-            self.writer.addNewLine()
-            for i in range(self.fabric.frameBitsPerRow):
-                self.writer.addInstantiation(
-                    "my_buf",
-                    f"data_inbuf_{i}",
-                    portsPairs=[("A", f"FrameData[{i}]"), ("X", f"FrameData_i[{i}]")],
-                )
-            for i in range(self.fabric.frameBitsPerRow):
-                self.writer.addInstantiation(
-                    "my_buf",
-                    f"data_outbuf_{i}",
-                    portsPairs=[
-                        ("A", f"FrameData_O_i[{i}]"),
-                        ("X", f"FrameData_O[{i}]"),
-                    ],
-                )
+        # buffer FrameData signals
```

Comment:
I am unsure if we should always add the buffer. I guess they won't be optimised away if they are not needed?

Reply:
Since other tiles with config bits will have them, I would opt to keep the buffers both for consistency and to ensure that the timing of the FrameData signals of all rows is the same in an ASIC.